compiler/modules/front.syntax/parseExpr.bal

```diff
@@ -340,7 +340,7 @@ function finishPrimaryExpr(Tokenizer tok, Expr expr, Position startPos) returns
         opPos = tok.currentStartPos();
         check tok.advance();
         Position qnamePos = tok.currentStartPos();
-        string name = check tok.expectIdentifier();
+        string name = check tok.expectMethodName();
```

Comment:
This isn't really right, because we are expecting either
- an identifier, or
- a special method name followed by `(`

So I don't think introducing `expectMethodName` is the right approach.